src/targets/Transpiler.52.ts

```diff
@@ -0,0 +1,7 @@
+import { LuaTranspiler, TranspileError } from "../Transpiler";
+import { TSHelper as tsHelper } from "../TSHelper";
+
+import * as ts from "typescript";
+
+export class LuaTranspiler52 extends LuaTranspiler {
```

Comment:
Lua 5.2 added the `bit32` library, this could be used to implement bit operations

Reply:
Sure, but we should probably do that in a different PR.

Reply:
Moved to #119